MobileSDK_To_AmplifySwift_Guidance.md

```diff
@@ -0,0 +1,162 @@
+# Migrating from AWS SDK for iOS to Amplify Swift
+
+This guide helps you migrate your iOS app from the legacy AWS SDK for iOS to the modern [Amplify Swift](https://docs.amplify.aws/gen1/swift/) library. 
+
+---
+
+## General Migration Notes
+
+- **Amplify Swift** is the recommended library for all new iOS/macOS/watchOS/tvOS development.
+- For any AWS service not yet supported by Amplify, you can use the [AWS SDK for Swift](https://github.com/awslabs/aws-sdk-swift) or reference [Swift SDK code examples](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/swift).
+- For IoT, migrate to the [AWS IoT Device SDK for Swift](https://github.com/aws/aws-iot-device-sdk-swift/blob/main/Package.swift).
+- Amplify will make a best-effort attempt to preserve user auth sessions during migration, but some users may need to re-authenticate.
+
+---
+
+## Category‑by‑category migration
+
+### Authentication
+
+| AWS Mobile SDK for iOS                                                                                          | Amplify Swift                                                                                                     |
+| --------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| [SignUp](https://docs.amplify.aws/gen1/swift/sdk/auth/working-with-api/#signup)                                 | [Sign Up](https://docs.amplify.aws/gen1/swift/sdk/auth/working-with-api/#signup)                                  |
```

Comment:
nit: [SignUp]
https://docs.amplify.aws/gen1/swift/build-a-backend/auth/enable-sign-in/#register-a-user
in second column